js/damage.js

```diff
@@ -603,6 +608,9 @@ function buildDescription(description) {
     if (description.isHelpingHand) {
         output += "Helping Hand ";
     }
+    if (description.isFriendGuard) {
```

Comment:
Before you squash, you put this in the wrong place. It should be near the Reflect and Light Screen message.